protostar/protostar_toml/protostar_toml_version_checker.py

```diff
@@ -0,0 +1,47 @@
+from protostar.protostar_exception import ProtostarException
+from protostar.protostar_toml.io.protostar_toml_reader import ProtostarTOMLReader
+from protostar.utils import VersionManager
+
+
+REFER_TO = (
+    "Please refer to https://github.com/software-mansion/protostar/releases, "
+    "in order to convert protostar.toml or upgrade protostar to a newer version."
+)
+
+
+class ProtostarTOMLVersionChecker:
+    def __init__(
+        self,
+        protostar_toml_reader: ProtostarTOMLReader,
+        version_manager: VersionManager,
+    ):
+        self._protostar_toml_reader = protostar_toml_reader
+        self._version_manager = version_manager
+
+    def run(self):
+        declared_version_str = self._protostar_toml_reader.get_attribute(
+            "config", "protostar_version"
+        )
+
+        if not declared_version_str:
+            raise ProtostarException(
+                "No protostar_version attribute available in protostar.toml.\n"
```

Comment:
`protostar_version` can exist in the `protostar.toml` but Protostar doesn't know how to load it

```suggestion
                "Couldn't load `protostar_version`.\n"
                "`protostar.toml` could've been created by a newer version of Protostar".
```